services/orchest-ctl/app/cli/main.py

```diff
@@ -112,11 +112,44 @@ def stop():
 
 
 @typer_app.command()
-def status():
+def status(
+    ext: bool = typer.Option(
+        False,
+        "--ext",
+        show_default=False,
+        help="Get extensive status information.",
+    ),
+):
     """
-    Get status of Orchest.
+    Get Orchest version.
```

Comment:
This probably got copied along from the version.

```suggestion
    Get status of Orchest.
```